test/unit_tests/braket/devices/test_local_simulator.py

```diff
@@ -310,6 +310,16 @@ def test_run_gate_model():
     task = sim.run(Circuit().h(0).cnot(0, 1), 10)
     assert task.result() == GateModelQuantumTaskResult.from_object(GATE_MODEL_RESULT)
 
+def test_run_batch():
+    bell = Circuit().h(0).cnot(0, 1)
+    device = LocalSimulator(dummy)
+    num_tasks = 10
+    circuits = [bell for _ in range(num_tasks)]
+    batch = device.run_batch(circuits, shots=10)
+
+    assert(len(batch.results()), num_tasks)
+    for x in batch.results():
+        assert task.results() == GateModelQuantumTaskResult.from_object(GATE_MODEL_RESULT)
```

Comment:
Could we update the test so that we can assert the ordering of the results?

Reply:
Sure I can; but keep in mind that python docs says that Pool.[map, starmap, imap] keep result order. (Unordered results are provided by Pool.imap_unordered)